modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/service/impl/ClientExtensionEntryServiceImpl.java

```diff
@@ -103,6 +104,15 @@ public ClientExtensionEntry deleteClientExtensionEntry(
 		return clientExtensionEntry;
 	}
 
+	@Override
+	public List<ClientExtensionEntry> getClientExtensionEntries(
+			long companyId, int start, int end)
+		throws PortalException {
+
```

Comment:
We'll need some permission check here or else servlet will become the place to get all client extensions. Not in this draft, but we'll need to do it at some point

Reply:
Shouldn't we simply use the remote service if we want checks? :thinking: